### DISCUSSION TITLE
How To create table through flutter code in supabase?

### DISCUSSION THREAD

**Discussion by SamarpanCoder2002** [supabase/supabase#2994] [Questions] [answered]
I want to make a chat application, where every user contains a table.. for that when new users register in the app... I have to create the table in supabase for himself/herself manually from flutter code only.... I did not found any reference for that... Please help me... It's urgent

**Comment by soedirgo** [comment, marked as answer]
Already answered here: https://github.com/supabase/supabase/discussions/2981

**Reply by SamarpanCoder2002** [reply]
I want to make a chat application, where every user contains a table.. for that when new users register in the app... I have to create the table in supabase for himself/herself manually from flutter code only... not from supabase UI or external Postgres client... Please help me.
